packages/toggle-switch/toggle-switch.scss

```diff
@@ -22,152 +56,143 @@ $inverted-disabled-color: #969696;
     height: $toggle-height;
     padding: 0;
     align-items: center;
+    height: $toggle-wrapper-height;
 
     background-color: transparent;
-    color: currentColor;
+    color: var(--toggle-text-color, $toggle-text-color);
     @include jkl-text-style("desktop/body");
+    @include motion("standard", "expressive");
+    transition-property: color;
 
-    &__help-label {
-        margin-top: $component-spacing--medium;
+    &__label {
+        display: inline-block;
+        width: 100%;
+        white-space: nowrap;
+        overflow-x: hidden;
+        text-overflow: ellipsis;
     }
 
-    &__slider {
-        border-radius: 99rem;
+    &__pill {
+        flex-shrink: 0;
         display: inline-block;
-        height: $toggle-height;
         position: relative;
-        width: $toggle-width;
-        margin-right: $component-spacing--small;
         pointer-events: none;
+        width: $toggle-width;
+        height: $toggle-height;
+        border-radius: 999px;
+        margin-left: $component-spacing--large;
+        background-color: var(--toggle-background-color, $toggle-background-color);
```

Comment:
Er det nødvendig å spesifisere fallback fargen?

Reply:
Dvs, hadde det vært ryddigere å spesifisert det i toppen? Så du får default, light og dark i toppen? Eller at vi har data-dark eller default(hvit)

Reply:
Er nok ikke sikkert at man må spesifisere fallback på standardvarianten slik det er nå heller. Hvis vi spesifiserer forskjellige properties for light og dark mode (f.eks. `--bg.color` og `--bg-color--dark`) i toppen vil vi jo også miste mye av funksjonaliteten vi får her. Jeg skal uansett undersøke litt mer! Synes også det hadde vært greit å slippe å spesifisere fallback der man kan 😊

Reply:
Det ser ut som `postcss-custom-properties` kun ser etter fallback-verdier definert i en ren `:root`-selektor, dvs at den ikke skjønner `:root, [data-theme="light"]`. Så hvis vi har tre selektorer (én for root, én for lyst tema, og én for mørkt tema) slipper vi å definere fallbacks for standardverdiene, men man får til gjengjeld to sett like variabeldefinisjoner..

I "legacy"-blokken for `--inverted` må vi uansett spesifisere fallback til å eksplisitt hente den mørke varianten, så jeg heller mot å la det stå slik det er nå overalt. Hvis vi senere dropper IE11 og deaktiverer plugin-en skader det jo ikke at det er definert fallbacks fra sass-variablene uansett, siden det likevel er de som definerer verdien til variablen i utgangspunktet.